src/runtime/thread_pool.cc

```diff
@@ -394,12 +397,34 @@ int TVMBackendParallelLaunch(
     FTVMParallelLambda flambda,
     void* cdata,
     int num_task) {
+#ifndef USE_OMP
```

Comment:
consider change the macro to be ```TVM_THREADPOOL_USE_OMP``` to be more project specific.  Add a default value likie other cases.

Reply:
fixed